### DISCUSSION TITLE
When something says 'v10 only' does it mean v10 and above?

### DISCUSSION THREAD

**Discussion by asharrahman02** [rnmapbox/maps#3476] [Q&A]
https://rnmapbox.github.io/docs/components/MapView

Many items in the doc say v10 only, do they mean v10 onward? I am on Mapbox v11.3.0 and my OnMapIdle does not do anything but onCameraChanged works fine.

`        "@rnmapbox/maps",
        {
          "RNMapboxMapsVersion": "11.3.0",
          "RNMapboxMapsDownloadToken": "..."
        }`

**Comment by mfazekas** [comment]
@icon2341 Yep v10 only is from the time when we've supported both v8 and v10. We'd need to remove most of those.
